### PR TITLE
hello world gitter8

### PR DESCRIPTION
**Pull request by MasseGuillaume**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,27 +1,7 @@
...
-CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
-EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
-PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
-PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
-LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
-NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
-SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+To the extent possible under law, the author(s) have dedicated all copyright and related
+and neighboring rights to this template to the public domain worldwide.
+This template is distributed without any warranty. See <http://creativecommons.org/publicdomain/zero/1.0/>.
```
Comment (MasseGuillaume) [inline LICENSE]: CC0 license

File: README.md [hunk 2]
```diff
@@ -1,177 +1,7 @@
-## Overview
+A [Giter8][g8] template for Scala-Native
 
-[![Join the chat at https://gitter.im/scala-native/scala-native](https://badges.gitter.im/scala-native/scala-native.svg)](https://gitter.im/scala-native/scala-native?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[![Build Status](https://travis-ci.org/scala-native/scala-native-example.svg?branch=master)](https://travis-ci.org/scala-native/scala-native-example)
-
-This project is an example of how you can employ [Scala Native] on your next project.
```
Comment (MasseGuillaume) [inline README.md]: I think we should keep this information in the doc.

File: src/main/scala/smallpt.scala [hunk 3]
```diff
@@ -1,208 +0,0 @@
-package demo
```
Comment (MasseGuillaume) [inline src/main/scala/smallpt.scala]: If this is a skeleton, it makes more sense to have a simple hello world. This is still available in the scala-native repository.

**Comment by densh** [comment]
CI is broken after this change.

File: src/main/scala/smallpt.scala (hunk 3 above)
Comment (densh) [inline src/main/scala/smallpt.scala]: I agree.

**Comment by densh** [comment]
Needs a rebase due to 0.2.0 release.

**Pull request closed without merging**